.github/workflows/pr-realm-js.yml

```diff
@@ -182,16 +182,7 @@ jobs:
         with:
           key: ${{ runner.os }}-${{ matrix.os }}-${{ matrix.arch }}
           max-size: '2.0G'
-
-      - name: Prepend ccache executables to the PATH
-        if: ${{ runner.os != 'Windows' }}
-        run: |
-          echo "/usr/lib/ccache:/usr/local/opt/ccache/libexec" >> $GITHUB_PATH
```

Comment:
The action will create symlinks in `/usr/local/bin`, which is why we no longer need to modify the path.